client/galaxy/scripts/components/Workflow/Editor/Node.vue

```diff
@@ -70,6 +82,14 @@ export default {
         canClone() {
             return this.type != "subworkflow";
         },
+        isEnabled() {
+            const Galaxy = getGalaxyInstance();
+            const isRecommendationEnabled = Galaxy.config.enable_tool_recommendations;
+            if (isRecommendationEnabled === true || isRecommendationEnabled === "true") {
```

Comment:
Is there a case in which this is a string?

Reply:
yes, "true" was not necessary.